gcp/compute_engine/cloudrun_serverless_neg_https_lb/variables.tf

```diff
@@ -0,0 +1,76 @@
+variable project {
+  description = "The project to deploy to"
+  type        = string
+}
+
+variable region {
+  description = "Location for serverless_neg and load balancer"
+  type        = string
+  default     = "europe-west-2"
+}
+
+variable serverless_neg_name {
+  description = "Name for serverless network endpoint group"
+  type        = string
+  default     = "serverless-neg"
+}
+
+variable cloudrun_service_name {
+  description = "Cloud run service name"
+  type        = string
+}
+
+variable serverless_https_lb_name {
+  description = "Name for the forwarding rule and prefix for supporting resources"
+  type        = string
+}
+
+variable serverless_https_lb_http_forward {
+  description = "Set to `false` to disable HTTP port 80 forward"
+  type        = bool
+  default     = false
```

Comment:
Don't we want this defaulting to true. What happens when it is true? Does it create a second forwarding rule?

Reply:
yes, it creates a second forwarding rule for http traffic